hyperspy/tests/signal/test_tools.py

```diff
@@ -920,6 +918,40 @@ def test_lazy_changetype_rechunk():
     assert chunks_newest == chunks_new
 
 
+def test_lazy_changetype_rechunk_False():
+    ar = da.ones((50, 50, 256, 256), chunks=(5, 5, 256, 256), dtype='uint8')
+    s = signals.Signal2D(ar).as_lazy()
+    s._make_lazy(rechunk=True)
+    assert s.data.dtype is np.dtype('uint8')
+    chunks_old = s.data.chunks
+    s.change_dtype('float', rechunk=False)
+    assert s.data.dtype is np.dtype('float')
+    assert chunks_old == s.data.chunks
+
+
+def test_lazy_reduce_rechunk():
```

Comment:
Use @pytest.mark.parametrize here for the list of method?

Reply:
I agree that this would lead to clearer test failures, but it's not straightforward here since the signal is created internally. As it does the job and there is not much time left, I suggest to leave it as it is for this release.